# Service to Jog Cartesians
float64 x
float64 y
float64 z
---
int64 ret
string msg